client/asset/dcr/dcr.go

```diff
@@ -59,6 +59,7 @@ var (
 type rpcClient interface {
 	SendRawTransaction(tx *wire.MsgTx, allowHighFees bool) (*chainhash.Hash, error)
 	GetTxOut(txHash *chainhash.Hash, index uint32, mempool bool) (*chainjson.GetTxOutResult, error)
+	GetBalanceMinConf(account string, minConfirms int) (*walletjson.GetBalanceResult, error)
```

Comment:
```
getbalance ("account" minconf=1)

Calculates and returns the balance of all accounts.

Arguments:
1. account (string, optional)             The account name to query the balance for, or "*" to consider all accounts (default="*")
2. minconf (numeric, optional, default=1) Minimum number of block confirmations required before an unspent output's value is included in the balance

Result:
{
 "balances": [{                         (array of object) Balances for all accounts.
  "accountname": "value",               (string)          Name of account.
  "immaturecoinbaserewards": n.nnn,     (numeric)         Immature Coinbase reward coins.
  "immaturestakegeneration": n.nnn,     (numeric)         Number of immature stake coins.
  "lockedbytickets": n.nnn,             (numeric)         Coins locked by tickets.
  "spendable": n.nnn,                   (numeric)         Spendable number of coins.
  "total": n.nnn,                       (numeric)         Total amount of coins.
  "unconfirmed": n.nnn,                 (numeric)         Unconfirmed number of coins.
  "votingauthority": n.nnn,             (numeric)         Coins for voting authority.
 },...],                                                  
 "blockhash": "value",                  (string)          Block hash.
 "totalimmaturecoinbaserewards": n.nnn, (numeric)         Total number of immature coinbase reward coins.
 "totalimmaturestakegeneration": n.nnn, (numeric)         Total number of immature stake coins.
 "totallockedbytickets": n.nnn,         (numeric)         Total number of coins locked by tickets.
 "totalspendable": n.nnn,               (numeric)         Total number of spendable number of coins.
 "cumulativetotal": n.nnn,              (numeric)         Total number of coins.
 "totalunconfirmed": n.nnn,             (numeric)         Total number of unconfirmed coins.
 "totalvotingauthority": n.nnn,         (numeric)         Total number of coins for voting authority.
}   
```

For DCR, calling this with `minconf=0`, we consider `"spendable"` as the Available balance, and `"immaturecoinbaserewards"` + `"totalimmaturestakegeneration"` + `"totallockedbytickets"` as immature, although the `totallockedbytickets` is debatable since it is not just immature, it needs to be spent by a vote or revoke before it can come back to the wallet.  But for the purposes of the DEX wallet, it may make sense to account for it as immature rather than ignore it.

Reply:
Bumps our `minNetworkVersion` to `190000`, and we lose litecoin which isn't there yet.

I was on 0.18. After building 0.20 (released a couple of weeks ago), I was seeing a `Fee estimation failed. Fallbackfee is disabled` error. I had to set `-fallbackfee=0.00001` in the calls to `bitcoind` in the harness.